packages/web-react/src/components/Grid/__tests__/Grid.test.tsx

```diff
@@ -49,4 +49,48 @@ describe('Grid', () => {
     const element = dom.container.querySelector('div') as HTMLElement;
     expect(element).toHaveClass('Grid--cols-2');
   });
+
+  it('should render with custom spacing', () => {
+    const dom = render(<Grid spacing="space-600" />);
```

Comment:
It would be better to target the element according to https://testing-library.com/docs/queries/about#priority. It's a bit tricky in this case though... it would probably be best to add testid

Reply:
https://github.com/lmc-eu/spirit-design-system/pull/1561#discussion_r1691320454